src/store/mesh/getters.ts

```diff
@@ -61,10 +68,20 @@ export const getters: GetterTree<MeshState, RootState> = {
       profiles.push({
         name,
         active: name === bedMesh.profile_name,
-        min,
-        max,
-        range: Math.abs(min - max),
-        ...profile
+        adaptive: false,
+        range: Math.abs(min - max)
+      })
+    }
+
+    if (!(bedMesh.profile_name in klipperProfiles)) {
```

Comment:
We need to make sure that there is an active mesh set (should be enough to check for `profile_name`)

```suggestion
    if (bedMesh.profile_name && !(bedMesh.profile_name in klipperProfiles)) {
```

Without that check, this will happen:

![image](https://github.com/fluidd-core/fluidd/assets/85504/0ad1b4e2-c59a-4903-8628-ce8fc780e549)

Reply:
Nice catch, thanks!